src/controller/model/endpoint.ts

```diff
@@ -602,19 +576,14 @@ class Endpoint extends Entity {
         commandID: number, status: number, clusterID: number, transactionSequenceNumber: number, options?: Options
     ): Promise<void> {
         assert(!options || !options.hasOwnProperty('transactionSequenceNumber'), 'Use parameter');
-        options = this.getOptionsWithDefaults(options, true, Zcl.Direction.SERVER_TO_CLIENT, null);
         const payload = {cmdId: commandID, statusCode: status};
-        const frame = Zcl.ZclFrame.create(
-            Zcl.FrameType.GLOBAL, options.direction, options.disableDefaultResponse,
-            options.manufacturerCode, transactionSequenceNumber, 'defaultRsp', clusterID, payload, options.reservedBits
-        );
-
         const log = `DefaultResponse ${this.deviceIeeeAddress}/${this.ID} ` +
             `${clusterID}(${commandID}, ${JSON.stringify(options)})`;
         debug.info(log);
```

Comment:
This log statement should go into `zclCommand()` which should fix https://github.com/Koenkk/zigbee-herdsman/pull/978/files#r1530963264

Reply:
So these statement can be removed and the type should be added in https://github.com/Koenkk/zigbee-herdsman/pull/978/files#diff-be0e9f494ec1a1e1e2c5d8c33866b7ad6b8e2f080c85133eced5c95a249e41bfR858

Reply:
Done, but we lost some readability of the debug logs.
was:
`Write 0x129/1 genOnOff({"onOff":1}, ...`
became:
`ZCL command 0x129/1 genOnOff.write([{"attrId":0,"attrData":1,"dataType":16}], ...`
because at the zclCommand level the data has already been converted

Reply:
True, could we add the error log as an optional argument to `zclCommand`?

Reply:
yes, can. I try it later

Reply:
Done.
was:
`Write 0x129/1 genOnOff({"onOff":1}, ...`
became:
`ZCL command 0x129/1 genOnOff.write({"onOff":1}, ...`